django_reference_app/settings.py

```diff
@@ -81,15 +84,16 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'django_reference',
-        'USER': os.getenv('POSTGRES_USER'),
-        'PASSWORD': os.getenv('POSTGRES_PASSWORD'),
-        'HOST': os.getenv('POSTGRES_HOST'),
+        'NAME': os.getenv('POSTGRES_DATABASE', 'django_reference_app'),
```

Comment:
The db name change isn't reflected in the Helm values and will break job-migration also.